Sniffs/UpdateURISniff.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace ET\ElegantThemes\Sniffs;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+
+class UpdateURISniff implements Sniff {
+
+	/**
+	 * Returns the token types that this sniff is interested in.
+	 * @return array(int)
+	 */
+	public function register() {
+		return [ T_COMMENT, T_DOC_COMMENT_STRING ];
+	}
+
+	/**
+	 * Processes the tokens that this sniff is interested in.
+	 *
+	 * @param File $phpcsFile                 The file where the token was found.
+	 * @param int  $stackPtr                  The position in the stack where
+	 *                                        the token was found.
+	 *
+	 * @return void
+	 */
+	public function process( File $phpcsFile, $stackPtr ) {
+        $tokens = $phpcsFile->getTokens();
+
+		if ( ! $this->isETUpdateURI( $tokens[ $stackPtr ]['content'] ) ) {
+			$phpcsFile->addError( "Invalid plugin update URI", $stackPtr, 'InvalidUpdateURI' );
+		}
+    }
+
+    public function isETUpdateURI( $content ) {
```

Comment:
@lanresmith Could we add docblock please?

Reply:
@ayubadiputra Added.